exercises/G-numbers/exercise.js

```diff
@@ -1 +1,5 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 28;
+var numberOfMentors = 10;
+var total = numberOfStudents- + numberOfMentors
```

Comment:
The minus operator here results in the wrong outcome. It outputs 18 when it should be 38